velero-plugins/configmap/restore.go

```diff
@@ -0,0 +1,39 @@
+package configmap
+
+import (
+	"github.com/konveyor/openshift-velero-plugin/velero-plugins/common"
+	"github.com/sirupsen/logrus"
+	"github.com/vmware-tanzu/velero/pkg/plugin/velero"
+	"k8s.io/apimachinery/pkg/api/meta"
+)
+
+// RestorePlugin is a restore item action plugin for Velero
+type RestorePlugin struct {
+	Log logrus.FieldLogger
+}
+
+// AppliesTo returns a velero.ResourceSelector that applies to configmaps
+func (p *RestorePlugin) AppliesTo() (velero.ResourceSelector, error) {
+	return velero.ResourceSelector{
+		IncludedResources: []string{"configmaps"},
+	}, nil
+}
+
+// Execute action for the restore plugin for the configmap resource
+// We want to exclude from backup configmaps that contain ownerReference to build pods
```

Comment:
would rewrite to be easier to read

> We want to skip restore of ConfigMaps that contain skip annotation

Reply:
resolved in 493dc5ca17eac6b27d0dc2c0556f18f0a1f2c521